ramalama/plugins/runtimes/inference/llama_cpp.py

```
@@ -494,28 +495,46 @@ def _serve_router(self, args: argparse.Namespace) -> None:
         engine.exec()
 
+    def serve_router_nonblocking(self, args: argparse.Namespace) -> None:
+        """Start the router-mode server non-blocking (for sandbox use)."""
+        args.detach = True
+        engine = self._build_router_engine(args)
+
+        if args.dryrun:
+            engine.dryrun()
+            return
+
+        subprocess.Popen(engine.exec_args)
```

Ruff (0.15.20)

[error] `subprocess` call: check for execution of untrusted input

(S603)